src/interface/storage.thrift

```diff
@@ -428,6 +428,14 @@ struct DropCPRequest {
     2: string                       name,
 }
 
+struct RebuildIndexRequest {
+    1: common.GraphSpaceID          space_id,
+    2: list<common.PartitionID>     parts,
+    3: common.SchemaID              schema_id,
+    4: common.SchemaVer             schema_version,
```

Comment:
I think , schema_id and schema_version can be delete, WDYT?

Reply:
To make it simple and cleanly, prefer save it in protocol and not judge in storage process.